### ISSUE TITLE
Add "find my county" button 

### ISSUE THREAD

**Issue by xunhuang** [xunhuang/covid-19#9]
Add "find my county" button on top and call geo api

**Comment by tumbleshack** [comment]
The geo API looks to be called automatically on page load at the moment... should this button replace the automatic call?

**Comment by xunhuang** [comment]
that was the original intent, before I found a cheaper alternative. Here is the dynamics

- Google's geo API is expensive (like 100x than other vendors)
- The other vendors is not as accurate. 

here how I envision geo ip works 
1) for normal traffic, route them to the cheaper alternative. 
2) for the button click, use google's API. Update the cookie after that. 

This is good for navigational change too, when you navigate too far away, clicking on this buttons gets you back home.

**Comment by tumbleshack** [comment]
I'd be happy to tackle this next if you're not already working on it. Looking to improve my understanding of React Context, which it looks like you use to keep track of the location.

Where do you envision this button? My initial instinct says to the left or the right of the search bar. (and shrink the bar's width a little bit)

**Comment by xunhuang** [comment]
Yes. I’d be grateful if you can take this one.  Yes left/right of the search bar would be perfect.

**Comment by tumbleshack** [comment]
Closed by #99
